README.md

```diff
@@ -29,6 +29,7 @@
 * [valita](https://github.com/badrap/valita)
 * [yup](https://github.com/jquense/yup)
 * [zod](https://github.com/vriad/zod)
+* [mojotech/json-type-validaton](https://github.com/mojotech/json-type-validation)
```

Comment:
Please:

1. Include the package in alphabetical order (but don't count the `@`)
2. However, please do include the `@` in the package name

In other words, `@mojotech/json-type-validaton` should be between `marshal` and `myzod`.